src/components/RewardsCard/index.tsx

```diff
@@ -25,22 +26,24 @@ export type RewardsCardProps = {
 
 function RewardsCard({
   rewardImage,
+  rewardType,
   claimsLeft,
-  decimals,
+  decimals = 0,
```

Comment:
this doesn't handle the null case, only the undefined case, so if decimals is null then null is passed to `getDecimalNumberFromAmount`. getDecimalNumberFromAmount doesn't handle the null case

Reply:
jsfiddle
![image](https://github.com/user-attachments/assets/17a2ed55-02d7-495e-bbf0-7f28df4cb525)

Reply:
Good catch